app/src/main/java/com/example/mytrainingpal/components/Navigation.kt

```diff
@@ -55,14 +55,18 @@ sealed class Screen(
 
     object Settings :
         Screen("settingsMain", "Settings", Icons.Default.Settings, RouteGroups.SETTINGS.route)
+    object Toggle :
+        Screen("toggle", "Toggle", Icons.Default.Settings, RouteGroups.TOGGLE.route)
+
 }
 
 enum class RouteGroups(val route: String) {
     HOME("home"),
     MUSCLE_PAIN("musclePain"),
     TRAINING("training"),
     CALENDAR("calendar"),
-    SETTINGS("settings")
+    SETTINGS("settings"),
+    TOGGLE("toggle")
```

Comment:
This should actually not be here. The toggle screen belongs to the training group
```suggestion
```